code/modules/client/preferences.dm

```diff
@@ -1230,16 +1254,22 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 							b_markings = hex2num(copytext(new_markings, 6, 8))
 
 				if("m_style")
-					if((species in list("Unathi", "Vulpkanin", "Tajaran"))) // Species with markings
+					if((species in list("Unathi", "Vulpkanin", "Tajaran", "Machine"))) // Species with markings
 						var/list/valid_markings = list()
 						for(var/markingstyle in marking_styles_list)
 							var/datum/sprite_accessory/M = marking_styles_list[markingstyle]
 							if( !(species in M.species_allowed))
 								continue
 
+							if(species == "Machine") //Species that can use prosthetic heads.
+								if(!("head" in rlimb_data) && !(M.name == "None")) //If the character can have prosthetic heads and they have the default head (with screen), no optic markings.

```

Comment:
extra brackets on the == condition

... yes, the in operator gets pissy if you don't use brackets around it, but the == operator never does, it's byond, what do you expect